lib/Data/Chronicle/Writer.pm

```diff
@@ -73,6 +74,12 @@ has [qw(cache_writer db_handle)] => (
     default => undef,
 );
 
+has 'ttl' => (
+    isa      => 'Int',
+    is       => 'ro',
+    required => 1
```

Comment:
I've did id 'required' because of Tom's 'Could we make this a parameter to the constructor, so that Chronicle::Writer->new throws an exception if it's not provided? '